src/service/extensions-impl.js

```diff
@@ -510,7 +522,9 @@ export class Extensions {
     // Install runtime styles.
     installStylesLegacy(
       childWin.document,
-      isExperimentOn(this.win, 'fie-css-cleanup')
+      isExperimentOn(this.win, 'fie-css-cleanup') &&
```

Comment:
let's extract a method to not dup the code here

Reply:
I could also remove `isExperimentOn` from the expression and only keep `getExperimentBranch`. It would return null if experiment not enabled.